src/app/beer_garden/app.py

```diff
@@ -183,38 +163,21 @@ def _startup(self):
         for helper_thread in self.helper_threads:
             helper_thread.start()
 
-        self.logger.info("Starting log reader...")
-        self.log_reader.start()
-
-        self.logger.debug("Starting entry points...")
-        for entry_point in self.entry_points:
-            entry_point.start(
-                context=self.context,
-                log_queue=self.log_queue,
-                events_queue=self.events_queue,
-            )
-            self.events_manager.register(
-                CallableProcessor(partial(entry_point.send_event))
-            )
+        self.logger.debug("Creating and starting entry points...")
+        self.entry_manager.create_all()
+        self.entry_manager.start()
 
         self.logger.debug("Loading all local plugins...")
         PluginManager.instance().load_new()
 
-        self.logger.debug("Starting all local plugins...")
-        PluginManager.instance().start_all()
-
         self.logger.debug("Starting local plugin process monitoring...")
         PluginManager.instance().start()
 
         self.logger.debug("Starting scheduler")
         self.scheduler.start()
 
-        try:
-            self.events_manager.put(
-                brewtils.models.Event(name=Events.BARTENDER_STARTED.name)
-            )
-        except RequestException:
-            self.logger.warning("Unable to publish startup notification")
+        self.logger.debug("Publishing startup event")
+        publish(Event(name=Events.GARDEN_STARTED.name))
 
```

Comment:
I just realized that in all of the places we try to publish an event we use  ```Events.EVENT_TYPE.name```. Should we just drop the ```.name```? Then we could do stuff like:
```
if event == event.EVENT_TYPE:
```
I don't foresee an scenario where we would not have access to the Event's object for this enumeration.

Reply:
Yeah, we really only need the *string name* when we go to serialize it. I'm good with this, less to type :smile: